io-engine/src/bdev/nexus/nexus_bdev.rs

```diff
@@ -1109,11 +1108,9 @@ impl<'n> BdevOps for Nexus<'n> {
         Reactor::block_on(async move {
             let self_ref = unsafe { &mut *self_ptr };
 
-            let n = self_ref
-                .children
-                .iter()
-                .filter(|c| c.state() == ChildState::Open)
-                .count();
+            // TODO: double-check interaction with rebuild job logic
+            // TODO: cancel rebuild jobs?
+            let n = self_ref.children.iter().filter(|c| c.is_healthy()).count();
```

Comment:
interesting, I guess we should keep the open here otherwise we won't close the children being rebuilt?

Reply:
It was closing only Open before, so I don't want to change that at this moment. We can potentially bump into something unexpected if we change it.

Reply:
Ok fair enough, maybe we can make this change later for develop branch only?

Reply:
I've added a ticket to cover this and other things: https://dcsw-bolt.atlassian.net/browse/B2-842